api/src/auth/drivers/ldap.ts

```diff
@@ -66,12 +65,9 @@ export class LDAPAuthDriver extends AuthDriver {
 			throw new InvalidProviderConfigError({ provider });
 		}
 
-		const clientConfig = typeof config['client'] === 'object' ? config['client'] : {};
-
-		this.bindClient = ldap.createClient({ url: clientUrl, reconnect: true, ...clientConfig });
```

Comment:
Since `ldapts` does not have the `reconnect: true` option is reconnecting something we'll have to manually do?